### PR TITLE
Install: tiny cleanups

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

File: src/shared/install.c [hunk 1]
```diff
@@ -1995,8 +1995,6 @@ static int install_info_symlink_wants(
                                 return install_changes_add(changes, n_changes, -EIDRM, dst, n);
                         else
                                 return install_changes_add(changes, n_changes, -EUCLEAN, dst, NULL);
-
-                        continue;
```
Comment (keszybz) [inline src/shared/install.c:1999]: No, this is wrong. `return` was added inadvertently in 94e7298d309fef7710174def820e9d38e512a086. We want to do the full loop here and collect any errors in `changes`.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: src/shared/install.c (hunk 1 above)
Comment (keszybz) [reply src/shared/install.c:1999]: We should fix this before the release. I added this to the milestone.

**Comment by yuwata** [comment]
@keszybz Updated. PTAL.

**Comment by keszybz** [comment]
LGTM.

**Pull request merged**